core/src/main/java/org/mskcc/oncotree/utils/CacheUtil.java

```diff
@@ -120,15 +129,14 @@ public void setDateOfLastCacheRefresh(Date date) {
 
     public void resetCache() throws FailedCacheRefreshException {
         logger.info("resetCache() -- refilling tumor types cache");
-        tumorTypes = new HashMap<>();
```

Comment:
This was wiping out our cache regardless of whether or not we could successfully query the information from TopBraid. Cache will now only be refreshed if we managed to get something back from TopBraid.